redaxo/src/core/lib/util/version.php

```diff
@@ -39,23 +39,24 @@ public static function split(string $version): array
      * @param string $comparator Optional comparator
      * @psalm-param null|'='|'=='|'!='|'<>'|'<'|'<='|'>'|'>=' $comparator
      *
-     * @return int|bool
+     * @return bool
      */
-    public static function compare(string $version1, string $version2, ?string $comparator = null)
+    public static function compare(string $version1, string $version2, ?string $comparator = '<')
     {
+        // bc
+        $comparator = $comparator ?? '<';
```

Comment:
`null` hat sich wie gesagt in php 7 auch vorher schon wie `<` verhalten.
Habe das Verhalten jetzt bloß explizit gemacht, und so auch für PHP 8 sichergestellt.